src/gui/guiFormSpecMenu.cpp

```diff
@@ -529,6 +542,63 @@ void GUIFormSpecMenu::parseScrollBar(parserData* data, const std::string &elemen
 	errorstream<< "Invalid scrollbar element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseScrollBarOptions(parserData* data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() > 0) {
+		for (unsigned int i = 0; i < parts.size(); i++) {
```

Comment:
Maybe `u32` instead of `unsigned int`.

Reply:
I don't think it's necessary to use this for a counter.

Reply:
Actually here you could just do something like `for ( const std::string &i : parts)`.

Reply:
I suppose you could.  C++ has many strange syntactical things you can do.

Reply:
A ranged for loop isn't that strange…